Invoking command in '/home/itolab-chotaro/HDD/SBIR-sit/ros_ws/src/lidar_udp_catch': PYTHONPATH=/home/itolab-chotaro/HDD/SBIR-sit/ros_ws/install/lidar_udp_catch/lib/python3.6/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/lidar_udp_catch build --build-base /home/itolab-chotaro/HDD/SBIR-sit/ros_ws/build/lidar_udp_catch/build install --prefix /home/itolab-chotaro/HDD/SBIR-sit/ros_ws/install/lidar_udp_catch --record /home/itolab-chotaro/HDD/SBIR-sit/ros_ws/build/lidar_udp_catch/install.log --single-version-externally-managed
Invoked command in '/home/itolab-chotaro/HDD/SBIR-sit/ros_ws/src/lidar_udp_catch' returned '0': PYTHONPATH=/home/itolab-chotaro/HDD/SBIR-sit/ros_ws/install/lidar_udp_catch/lib/python3.6/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/lidar_udp_catch build --build-base /home/itolab-chotaro/HDD/SBIR-sit/ros_ws/build/lidar_udp_catch/build install --prefix /home/itolab-chotaro/HDD/SBIR-sit/ros_ws/install/lidar_udp_catch --record /home/itolab-chotaro/HDD/SBIR-sit/ros_ws/build/lidar_udp_catch/install.log --single-version-externally-managed
